packages/workflow/src/internals.ts

```diff
@@ -12,6 +12,7 @@ import {
   Workflow,
   WorkflowQueryType,
   ApplicationFailure,
```

Comment:
some lint error here you may want to take care of

```suggestion
```

Reply:
I'll fix in the next PR